docker/build_docker.sh

```diff
@@ -71,22 +71,19 @@ function push_to_hub {
     docker_with_torchacc_suffix="-"$(date +"%y%m%d")
 
     if [ "$push_to_hub" = true ]; then
-        for region in ${regions}
-        do
-            if [ -n "${DOCKER_USERNAME+x}" ] && [ -n "${DOCKER_PASSWORD+x}" ]; then
-                docker login --username=$DOCKER_USERNAME --password=$DOCKER_PASSWORD registry.cn-${region}.aliyuncs.com
-
-                # Push to pai-dlc as latest
-                docker tag ${docker_with_torchacc} registry.cn-${region}.aliyuncs.com/pai-dlc/${docker_with_torchacc}
-                docker push registry.cn-${region}.aliyuncs.com/pai-dlc/${docker_with_torchacc}
-
-                # Push to pai-dlc with date
-                docker tag ${docker_with_torchacc} registry.cn-${region}.aliyuncs.com/pai-dlc/${docker_with_torchacc}${docker_with_torchacc_suffix}
-                docker push registry.cn-${region}.aliyuncs.com/pai-dlc/${docker_with_torchacc}${docker_with_torchacc_suffix}
-            else
-                echo "No docker login information found. Skip uploading to docker hub."
-            fi
-        done
+        if [ -n "${DOCKER_USERNAME+x}" ] && [ -n "${DOCKER_PASSWORD+x}" ]; then
+            docker login --username=$DOCKER_USERNAME --password=$DOCKER_PASSWORD pai-platform-registry.cn-beijing.cr.aliyuncs.com
+
+            # Push to pai-dlc as latest
+            docker tag ${docker_with_torchacc} pai-platform-registry.cn-beijing.cr.aliyuncs.com/pai/${docker_with_torchacc}
+            docker push pai-platform-registry.cn-beijing.cr.aliyuncs.com/pai/${docker_with_torchacc}
+
+            # Push to pai-dlc with date
+            docker tag ${docker_with_torchacc} pai-platform-registry.cn-beijing.cr.aliyuncs.com/pai/${docker_with_torchacc}${docker_with_torchacc_suffix}
```

Comment:
Using Docker image tag to manage the relasing version?

Reply:
Yes, we upload the Docker image with the date information, which is defined in LINE 71. The variable `docker_with_torchacc` is defined in `docker/config.ini`.